extension/src/popup/ducks/accountServices.ts

```diff
@@ -238,15 +238,17 @@ interface InitialState {
   applicationState: APPLICATION_STATE;
   hasPrivateKey: boolean;
   publicKey: string;
+  connectingWalletType: string;
```

Comment:
pretty small - but should the type here be `WalletType`? Would make it clear that we're looking for something from that enum here

Reply:
ya  good call https://github.com/stellar/freighter/pull/531/commits/b1bba6d7b161390d23d023bc7ad0fd87021d1dec